docs/setup/pip.md

```diff
@@ -2,13 +2,13 @@
 
 Opting for `pip` is ideal for those with a comprehensive grasp of Python's ecosystem and an understanding of operating system specifics. This approach, while offering adaptability for custom setups, necessitates knowledge of environment setup and dependency handling.
 
-The `khiops` binary must be installed as a prerequisite, which also ensures the installation of the appropriate version of `MPICH` required for the library.
+The Khiops binaries must be installed as a prerequisite. This also ensures the installation of the appropriate version of `MPICH` library.
 
-We support :simple-python: **Python from 3.8 to 3.11**. For users on :simple-kaggle: **Kaggle Notebooks** or :simple-googlecolab: **Google Colaboratory**, we have tested and recommend using the :simple-anaconda: **Conda** package for installing Khiops.
+We support :simple-python: **Python from 3.8 to 3.11**. 
```

Comment:
the comment about colab and Kaggle was a gentle reminder for those who might have opted for "pip" and missed the information => "please opt for the conda option instead". What do you think ?

Reply:
IMHO, as long as we state from the outset (on the index page) that Colab / Kaggle needs Conda install, this makes it clear (to me) that this puts Pip out of the discussion for these environments.

Reply:
OK with it

Reply:
Yes, OK